all_test.go

```diff
@@ -105,6 +109,80 @@ func TestGovulncheck(t *testing.T) {
 	rungo(t, "run", "golang.org/x/vuln/cmd/govulncheck@latest", "./...")
 }
 
+func TestExportedSymbolsHaveDocs(t *testing.T) {
+	// TODO(https://github.com/googleapis/librarian/issues/522): turn on once
+	// all existing symbols have docs.
+	t.Skip()
+
+	err := filepath.WalkDir(".", func(path string, d os.DirEntry, err error) error {
+		if err != nil || d.IsDir() || !strings.HasSuffix(path, ".go") || strings.HasSuffix(path, "_test.go") {
```

Comment:
Don't you want to return a non-nil error?

Reply:
Done